src/app/pages/home/home.page.ts

```diff
@@ -53,12 +81,64 @@ export class HomePage implements OnInit {
     private readonly diaBackendAuthService: DiaBackendAuthService,
     private readonly diaBackendAssetRepository: DiaBackendAssetRepository,
     private readonly diaBackendTransactionRepository: DiaBackendTransactionRepository,
-    private readonly pushNotificationService: PushNotificationService,
-    private readonly ignoredTransactionRepository: IgnoredTransactionRepository
+    private readonly ignoredTransactionRepository: IgnoredTransactionRepository,
+    private readonly imageStore: ImageStore,
+    private readonly httpClient: HttpClient
   ) {}
 
   ngOnInit() {
-    this.pushNotificationService.configure();
+    /**
+     * TODO: Remove this ugly WORKAROUND by using repository pattern to cache the expired assets and proofs.
+     */
+    this.pollingTransaction$()
```

Comment:
@shc261392 I don't think we need to handle the possible network error in this release as this is just an ugly workaround. Ideally, network errors should be handled and display a snackbar for all `DiaBackend**` services. However, there is no such implementation in current stage. After all, this app cannot display any icon if users have network connectivity problem. We should take care such error after we take offline availability into account. See [the project](https://github.com/numbersprotocol/capture-lite/projects/3). Also, if we catch everything during the polling, which requires multiple `catchError` operators in different places of this pipe, it would be difficult to spot bugs caused by other internal exceptions in the future (before rewrite this workaround).

Ultimately, we need to refactor the data flow of `DiaBackendAsset` and `Proof` by caching the backend asset with repository pattern. Every error regarding IO could be handled within the repository instead of UI components.